inline_verifier.go

```diff
@@ -91,6 +91,9 @@ func (v *InlineVerifier) CheckFingerprintInline(tx *sql.Tx, targetDb, targetTabl
 		targetFingerprints[pk] = rowData[1]
 		targetDecompressedData[pk] = make(map[string][]byte)
 
+		// Note that the FingerprintQuery returns the columns: pk, fingerprint,
+		// compressedData1, compressedData2, ...
+		// If there are no compressed data, only 2 columns are returned.
```

Comment:
nit: `If there are no compressed data, only 2 columns are returned and this block will be skipped`

Reply:
I'll change this in the next PR as I moved this code around.